cdk/visit/lambda_code/log_visit/log_visit.py

```diff
@@ -38,10 +55,15 @@ def __init__(self, table, ses_client):
             self.client = ses_client
 
     def checkRegistration(self, current_user):
-        response = self.visits.query(
-            KeyConditionExpression=Key('PK').eq(current_user)
+        print("Checking registration for: " + current_user)
```

Comment:
For all reads- we should be reading from the old table, right?

Reply:
Yep! Thanks for the catch!